src/widgets/NamedEntitiesWidget/NamedEntitiesWidget.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="named-entities">
```

Comment:
We're missing a way to clear the selection from the widget:

From 
![](https://github.trello.services/images/mini-trello-icon.png) [As a user, I can see a list of named entities in the passage](https://trello.com/c/WPEGWVfP)
> Have a way to clear the selection in the widget (to see all the entities)

![image](https://user-images.githubusercontent.com/629062/82500377-e089ab00-9ab8-11ea-9aea-a14271946e54.png)

Reply:
de-selection is also referenced in ![](https://github.trello.services/images/mini-trello-icon.png)[As a user, I can select a named entity in the passage and update the named entities widget to see more detail](https://trello.com/c/oDQlPPBA).

A "clear" option in the widget would be nice, but in addition I'm open to something with a modifier key like we have in the "highlight" mode on [scaife.perseus.org](https://scaife.perseus.org/reader/urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:1.1-1.7/?highlight=%40%CE%A0%CE%B7%CE%BB%CE%B7%CF%8A%CE%AC%CE%B4%CE%B5%CF%89%5B1%5D).

(I'm _not_ proposing a highlight mode, but rather a way to clear the selection without having to click a "clear" button in the widget).

Reply:
To conserve space in the widget, I just made it a toggle so if you click on the entity again it unselects it.  I could put a little X in top right of the "well" if that is more intuitive.

Reply:
The `X` helps; thanks.